btservice/AndroidBluetoothServer.cpp

```diff
@@ -15,41 +15,6 @@ AndroidBluetoothServer::AndroidBluetoothServer(openauto::configuration::IConfigu
     , handshakeState_(ConnectionStatus::IDLE)
```

Comment:
@icecube45 , this raises a compilation error on my side since the member _handshakeState__ was removed from the class definition.
`openauto/btservice/AndroidBluetoothServer.cpp: In constructor ‘openauto::btservice::AndroidBluetoothServer::AndroidBluetoothServer(openauto::configuration::IConfiguration::Pointer)’:
openauto/btservice/AndroidBluetoothServer.cpp:15:7: error: class ‘openauto::btservice::AndroidBluetoothServer’ does not have any field named ‘handshakeState_’
   15 |     , handshakeState_(ConnectionStatus::IDLE)
      |       ^~~~~~~~~~~~~~~
openauto/btservice/AndroidBluetoothServer.cpp:15:23: error: ‘ConnectionStatus’ has not been declared
   15 |     , handshakeState_(ConnectionStatus::IDLE)
      |                       ^~~~~~~~~~~~~~~~
`

Reply:
Oh yes that shouldn't be there - surprised mine compiled, maybe i didnt push everything. I'll pull a fresh clone and make sure it's all good

Reply:
Oh yes that shouldn't be there - surprised mine compiled, maybe i didnt push everything. I'll pull a fresh clone and make sure it's all good